CenterForOpenScience/ember-osf-web
# [ENG-4488] Upgrade ember-cli-clipboard and ember-get-config

-   Ticket: [ENG-4488]
-   Feature flag: n/a

## Purpose

Upgrade ember-cli-clipboard, which was doing deprecated things

## Summary of Changes

1. Upgrade ember-cli-clipboard
2. Modify usage of CopyButton to match new API
3. Upgrade ember-get-config to try (unsuccessfully) to fix broken local tests

## Side Effects

Let's see if this breaks tests in CI, or whether it's just locally.

## QA Notes

Make sure various copy buttons work
* Dev Apps/Tokens
* File detail versions hashes
* File list embed codes


[ENG-4488]: https://openscience.atlassian.net/browse/ENG-4488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
